app/src/main/java/com/example/weather/ui/temp/TempFragment.java

```diff
@@ -58,4 +93,11 @@ public void onClick(View view) {
             }
         });
     }
-}
\ No newline at end of file
+
+    @Override
+    public void onPause() {
+        super.onPause();
+        viewModel.update();
```

Comment:
![image](https://user-images.githubusercontent.com/42149893/159153153-a369e53c-a91f-4eeb-b272-6499c82e426b.png)

Reply:
This is genius move for TempView!!! I'll explain it at the university